integration/spark/src/test/scala/org/apache/carbondata/geo/GeoTest.scala

```diff
@@ -82,6 +81,33 @@ class GeoTest extends QueryTest with BeforeAndAfterAll with BeforeAndAfterEach {
       case None => assert(false)
     }
   }
+  test("test geo table drop spatial index column") {
+    createTable()
+    loadData()
+    val exception = intercept[MalformedCarbonCommandException](sql(s"alter table $table1 drop columns(mygeohash)"))
+    assert(exception.getMessage.contains(
```

Comment:
some more test case can be added with mixed case column names for other table property validation added, example range column, bucket column etc

Reply:
ok done.